.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - init
       - run:
           name: helmfile dry-run
-          command: helmfile -e ${ENVIRONMENT_VALUES} -f helmfile-core.yaml apply --args --dry-run --suppress-secrets --detailed-exitcode=false
+          command: helmfile -e ${ENVIRONMENT_VALUES} -f helmfile-core.yaml template --validate
```

Comment:
how this differs from the previous version? do you mind explaining how this impacts our pipeline? thanks

Reply:
In the newer version of helmfile the dry-run has been removed.
Recommended validation method is now template --validate as it interactively acts upon an existing cluster, validation level is supposed to be deeper as I understand it